compiler/src/dmd/traits.d

```diff
@@ -1676,7 +1676,7 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
             uint errors = global.startGagging();
             Scope* sc2 = sc.push();
             sc2.tinst = null;
-            sc2.minst = null;
+            sc2.minst = null;   // this is why code for these are not emitted to object file
```

Comment:
I'm not finding where Scope is declared via GH and I've forgotten where it's located, but wouldn't it be good to also document this behavior there as well?

Reply:
Why what is emitted and what isn't took me a while to find out. It's more complex than just minst. This is the only case I found that set minst to null. It could use a refactor, but I thought I'd leave a breadcrumb for the next guy wondering why traits compiles doesn't emit code.

Reply:
Essentially, too many people have worked on this aspect and the result is incoherence.

Reply:
Indeed. Knowing what state the AST is in, is for sure an annoying beast!